src/utils.py

```diff
@@ -186,19 +187,23 @@ def run(self, weaviate_client: weaviate.Client):
             )
 
             # run indexing pipeline
-            self.run_indexing_pipeline(run_count, indexing_config, indexing_pipeline)
-
+            self.runner.run(indexing_pipeline)
+            
             check_weaviate_class_exists(
                 weaviate_client,
                 indexing_config["weaviate_class"],
             )
+            
+            logger.info(
```

Comment:
This should probable be above line 190 before we start the run

Reply:
makes sense